query-coordinator-http/src/main/resources/reference.conf

```diff
@@ -58,6 +58,7 @@ com.socrata.query-coordinator {
   get-schema-timeout = 10s
   query-timeout = 2m
   all-secondary-instance-names = [pg.primus]
+  secondary-selector-map-max-capacity = 2000
```

Comment:
Also, I'd like this to be called something like "secondary-selector-max-cache-entries", since that makes it clearer what's going on.  

Right now at a glance it looks like it could be talking about the maximum amount of RAM.